src/platform/iotjs_module_i2c-linux-general.inl.h

```diff
@@ -134,6 +136,16 @@ int I2cSmbusAccess(int fd, uint8_t read_write, uint8_t command, int size,
 }
 
 
+int I2cSmbusReadByte(int fd) {
+  I2cSmbusData data;
+  int result;
+
+  result = I2cSmbusAccess(fd, I2C_SMBUS_READ, I2C_NOCMD, I2C_SMBUS_BYTE, &data);
+
+  return result >= 0 ? 0x0FF & data.byte : -1;
```

Comment:
I think 0xFF is just for masking so not needed to be maintained as literals. @LaszloLango Is it necessary ?